cratedb_toolkit/cfr/cli.py

```diff
@@ -127,6 +128,40 @@ def job_statistics_view(ctx: click.Context):
     jd(response)
 
 
+@make_command(job_statistics, "report", "View job statistics per report.")
+@click.pass_context
+def job_statistics_report(ctx: click.Context):
+    """
+    View job statistics about collected queries per report.
+    """
+    import cratedb_toolkit.cfr.marimo
+
+    address = DatabaseAddress.from_string(ctx.meta["cratedb_http_url"] or ctx.meta["cratedb_sqlalchemy_url"])
+    os.environ["CRATEDB_SQLALCHEMY_URL"] = address.dburi
+    cratedb_toolkit.cfr.marimo.app.run()
```

Comment:
Dear @WalBeh, this is another way of invoking a Marimo application, just by calling the `run()` method on the `app` instance. I guess it will not start a server to serve the notebook, but just execute it, and display its output on stdout.